pkg/bucketeer/user.go

```diff
@@ -31,3 +32,13 @@ func (u *User) Valid() bool {
 	}
 	return true
 }
+
+// NewUser creates a new User.
+//
+// id is mandatory and attributes is optional.
+func NewRestUser(id string, attributes map[string]string) *user.User {
```

Comment:
To prevent breaking change, I defined NewRestUser here. In the next PR, I'll rename `NewRestUser` to `NewUser`.